src/main/java/org/folio/services/email/impl/MailServiceImpl.java

```diff
@@ -68,17 +70,17 @@ public void sendEmail(JsonObject smtpConfigurationJson, JsonObject emailJson,
       String emailId = emailEntity.getId();
       long start = currentTimeMillis();
 
-      logger.info("Sending email {}: attempt {}/{}",
+      logger.info("sendEmail:: Sending email {}: attempt {}/{}",
         emailId, emailEntity.getAttemptCount() + 1, RETRY_MAX_ATTEMPTS);
 
       defineMailClient(mailConfig)
         .sendMail(mailMessage)
-        .onSuccess(r -> logger.info("Email {} sent in {} ms", emailId, currentTimeMillis() - start))
-        .onFailure(t -> logger.error("Failed to send email {}: {}", emailId, t.getMessage()))
+        .onSuccess(r -> logger.info("sendEmail:: Email {} sent in {} ms", emailId, currentTimeMillis() - start))
+        .onFailure(t -> logger.error("sendEmail:: Failed to send email {}: {}", emailId, t.getMessage()))
         .map(emailJson)
         .onComplete(resultHandler);
     } catch (Exception ex) {
-      logger.error(format(ERROR_SENDING_EMAIL, ex.getMessage()));
+      logger.warn("sendEmail:: {}", format(ERROR_SENDING_EMAIL, ex));
```

Comment:
maybe exception with stacktrace should be logged as well?

Reply:
done